DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1

```diff
@@ -590,9 +590,13 @@ function Test-WebsiteBindings
     # Assume bindings do not need updating
     $BindingNeedsUpdating = $false
 
-    Write-Verbose "Getting Web Binding: $name"
+    <#
+        Currently there is a problem in the LCM where Get-WebBinding short circuts the verbose stream
+        Write-Log can be changed to add the -File switch to log to a directory for troubleshooting.
+        However it's pretty noisy so it's being left in but commented out.
+    #>
+
```

Comment:
What do you mean by Get-WebBinding short circuts the verbose stream?

Also, if we add -File switch it's not noisy anymore right? Perhaps we should do it, I don't think leaving this code commented out is a good idea - let's either work around it or remove it completely.